.gitignore

```diff
@@ -48,6 +48,7 @@ autom4te.cache/
 /vendor
 /ide/
 
+.zephir/
```

Comment:
Could you remove `.temp/` above. It is from Zephir v0.10

Reply:
Done